composer.json

```
@@ -25,8 +25,9 @@
     "extra": {
     },
     "require": {
-        "php": "^7.1",
+        "php": "^7.3 || ~8.0.0",
         "ext-json": "*",
+        "composer/package-versions-deprecated": "1.11.99.1",
```

```suggestion
        "composer/package-versions-deprecated": "^1.11",
```

Please use a constraint to avoid possible conflicts in the future.